napari_aicsimageio/core.py

```diff
@@ -57,53 +150,27 @@ def reader_function(
 
     # Open file and get data
     img = AICSImage(path)
-    print(
-        f"AICSImageIO: Image contains {len(img.scenes)} scenes. "
-        f"napari-aicsimageio currently only supports loading first scene, "
-        f"will load scene: '{img.current_scene}'."
-    )
-
-    data = _get_full_image_data(img, in_memory=in_memory)
 
-    # Catch None data
-    if data is None:
-        return None
+    # Check for multiple scenes
+    if len(img.scenes) > 1:
+        print(
+            f"AICSImageIO: Image contains {len(img.scenes)} scenes. "
+            f"Supporting more than the first scene is a work in progress. "
+            f"Select a scene from the list widget. There may be dragons!"
+        )
```

Comment:
Hah. Love it.

Reply:
Changed wording to experimental, to better reflect the state 😄